pkg/cmd/release/download/download.go

```diff
@@ -224,6 +224,7 @@ func downloadAssets(dest *destinationWriter, httpClient *http.Client, toDownload
 		go func() {
 			for a := range jobs {
 				results <- downloadAsset(dest, httpClient, a.APIURL, a.Name, isArchive)
+				io.UpdateProgressBar(1)
```

Comment:
We could make the progress bar more precise by calculating the amount of data to be downloaded in `toDownload`, and then updating it as we receive data. In this PR, I chose to just update the progress bar as each individual asset is downloaded. Open to opinions!